crates/sp-lightclient/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
+name = "sp-lightclient"
+version = "0.1.0"
+authors = ["Vedhavyas Singareddi <ved@subspace.network>"]
+edition = "2021"
+license = "Apache-2.0"
+homepage = "https://subspace.network"
+repository = "https://github.com/subspace/subspace"
+description = "Light client substrate primitives for Subspace"
+include = [
+    "/src",
+    "/Cargo.toml",
+    "/README.md",
+]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "3.1.2", default-features = false }
+scale-info = { version = "2.1.1", default-features = false, features = ["derive"] }
+schnorrkel = { version = "0.9.1", default-features = false, features = ["u64_backend"] }
+sp-api = { version = "4.0.0-dev", default-features = false, git = "https://github.com/subspace/substrate", rev = "b49d8ee54f4bd6e4b5b48f45d08f75705fad95b4" }
+sp-arithmetic = { version = "5.0.0", default-features = false, git = "https://github.com/subspace/substrate", rev = "b49d8ee54f4bd6e4b5b48f45d08f75705fad95b4" }
+sp-consensus-subspace = { version = "0.1.0", path = "../sp-consensus-subspace", default-features = false }
+sp-std = { version = "4.0.0-dev", default-features = false, git = "https://github.com/subspace/substrate", rev = "b49d8ee54f4bd6e4b5b48f45d08f75705fad95b4" }
+subspace-core-primitives = { version = "0.1.0", path = "../subspace-core-primitives", default-features = false }
+subspace-solving = { version = "0.1.0", path = "../subspace-solving", default-features = false }
+subspace-verification = { version = "0.1.0", path = "../subspace-verification", default-features = false }
+
+[dev-dependencies]
+sp-runtime = { version = "6.0.0", git = "https://github.com/subspace/substrate", rev = "b49d8ee54f4bd6e4b5b48f45d08f75705fad95b4" }
```

Comment:
If you move `sp-runtime` into regular dependencies, you can remove `sp-api` as unnecessary.